.unreleased/pr_7455

```diff
@@ -0,0 +1 @@
+Implements: #7455: Support DROP NOT NULL on compressed hypertables
```

Comment:
This changelog is misleading because we are actually not supporting it on compressed hypertables.

I also think we shouldn't have changelog entries for every PR. This is a change to unreleased code (Hypercore TAM), and I don't think it makes sense to detail every change we do to Hypercore TAM before it is released.

Reply:
I didn't realize it was from the other commit.